pkgs/development/libraries/webp-pixbuf-loader/default.nix

```diff
@@ -51,13 +50,11 @@ stdenv.mkDerivation rec {
       --replace "@bindir@/gdk-pixbuf-thumbnailer" "$out/bin/webp-thumbnailer"
   '';
 
-  preInstall = ''
-    # environment variables controlling loaders.cache generation by gdk-pixbuf-query-loaders
+  postInstall = ''
     export GDK_PIXBUF_MODULE_FILE="$out/${loadersPath}"
     export GDK_PIXBUF_MODULEDIR="$out/${moduleDir}"
-  '';
+    gdk-pixbuf-query-loaders --update-cache
```

Comment:
Now that these variables are not used by the build system, we should probably limit their scope just to the query-loaders call.